### PR TITLE
New feature

### PR DESCRIPTION
**Pull request by konradmmm**
second draft

### CODE REVIEW

File: SquareV2.py [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        print("Empty File")
+        sys.exit(1)
+
+    elif not file.endswith(".json"):
+        print("not a JSON File")
+        sys.exit(1)
+    else:
+        return f
+#error handling if keys are incorrect
+def incorrect_keys(data):
```
Comment (Jasio10) [inline SquareV2.py:95]: what about adding x or y as not numeric  "x": "a"

File: SquareV2.py [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+incorrect_keys(data)
+
+#Sorted list of x and y coordinates to figure out best points for square corners
+sorted_x_coor_list = bubblesort(unpack(data, X_KEY))
+sorted_y_coor_list = bubblesort(unpack(data, Y_KEY))
+
+#create the square
+Square = create_Square(sorted_x_coor_list,sorted_y_coor_list)
+print("\ncoordinates inputed: ")
+for z in data:
```
Comment (Jasio10) [inline SquareV2.py]: what is z in data

File: SquareV2.py [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+    print(data[z])
+
+print("\nThe smallest square that will fit your coordinates has the following coordinates: ")
+for key, value in Square.items():
+    print(key, ': ', value)
+
+
+
```
Comment (Jasio10) [inline SquareV2.py:151]: Generally you are on the right track as per my knowledge.

File: SquareV2.py (hunk 2 above)
Comment (konradmmm) [reply SquareV2.py]: I updated z to be named coordinates to make more readable

File: SquareV2.py (hunk 1 above)
Comment (konradmmm) [reply SquareV2.py:95]: I adjusted the code to handle non numeric entries

**Review by Jasio10 — APPROVED** [review]
No bad, it works 
We can talk a little more about it on Saturday 
good job

**Pull request closed without merging**